index.html

```diff
@@ -318,6 +318,10 @@ <h3>New books</h3>
                             <li><a href="https://www.infoq.com/minibooks/jhipster-4x-mini-book" target="_blank">The JHipster Mini-Book 4.5</a> by Matt Raible. Now with a chapter on microservices! </li>
                             <li><b>Full Stack Development with JHipster</b> by Deepu K Sasidharan and Sendil Kumar. Get it on <a href="https://www.packtpub.com/application-development/full-stack-development-jhipster" target="_blank">Packt</a> and <a href="https://www.amazon.com/Stack-Development-JHipster-Deepu-Sasidharan/dp/178847631X" target="_blank">Amazon</a>.</li>
                         </ul>
+                        <h3>Free Collaborative learning</h3>
+                        <ul>
+                            <li>Learn by doing with <a href="www.jhipsterpresss.com" target="_blank">JhipsterPress</a>. Join a live collaborative project!</li>
```

Comment:
This is just another training so should be in the training section above. The site doesnt seem to work at the moment, so i'm not in favor of adding broken links to home page

Reply:
Hi Deepu,

JhipsterPress works with a Heroku free account, so it might take a minute to wake up (it works, believe me, I followed your GREAT book ;-) 

https://youtu.be/PelM_MyACkM 

Jhipster is a great tool, but it does not have a great product to show what your great tool can achieve!

You are a project leader, so you might know better than I do, but to me the idea is not just another training. Your book was great and the other training material good (sometimes), but not good enough because that material does not cover how to do a great product like WordPress. Maybe it is enough for experienced programmers like you that already know everything about Jhipster, but not for unexperienced ones like me (believe me on that). How many unexperienced programmers have drop on Jhipster because the tools is too difficult for them (because a lack of a great product as an example)? Do you have an statistic on that?

I wish you have a community of non-experienced programmers working together and trying to make a flagship project like JhipsterPress to see how the different parts work together in a real open source project that is as useful as a WordPress but made with Jhipster https://github.com/Tonterias/jhipsterpress10 ( version 10! v11 comming soon) with 20+ tips (example; https://github.com/Tonterias/jhipsterpress10/blob/master/ReadMe/Solution21.md) Take a look at the homepage on GitHub. That would have saved me a lot of time and helped me a lot!!!!! 

I'm not a programmer and I was able to create JhipsterPress. How? Thanks to SUPER Jon Rudell that helped me getting out of the problems I made when creating JhipsterPress. I think that it could be an interesting collaborative project that could become more complex and richer!

If you are interested I could explain what I think that would make it more exciting in the future, but do not tell me it is just another... you do not know what I have in my mind!

Have a nice day and thank you very much for your book. It is really great, as the starting point! 

Mike

PD: I understand your negative judgement, you must be really busy. Tell me where is that training section and I will put it there.

Reply:
@Tonterias I didn't mean to demotivate you. When I meant just another training, it wasn't a negative comment, all the training we have are great and I consider this as another kind of training material that's all.
To have it as a new section it needs to be very clear, you say free collaborative learning. What does that mean, how will someone collaborate on this to learn, is it a playground like JSFiddle or something for example. So in order to have a separate section, you need to justify what it means. 

Even if I agree to have it as a section, I can't agree to put a broken link `www.jhipsterpresss.com` doesn't work for me, I don't understand the "it takes a minute to wake up" comment? even if you host it using Heroku free, the link should be up all the time. It can take some time to load but shouldn't be 404, see below
![image](https://user-images.githubusercontent.com/1107223/49330859-cababa00-f594-11e8-8814-4cdd6cf56f38.png)

Reply:
Hi Deepu,

I do not know why, but it works for me though it takes a minute to wake up (and I haven't changed anything at heroku and it is ok now), though friends that use Apple are having problems too. 

I will ask Jon, when I finish the current version.

Anyway, a free collaborative project means that programmers that want to learn Jhipster can join  JHipsterPress project and learn by helping to develop the Wordpress made with Jhipster. JHipsterPress is monolithic by now, but when I finish the monolithic version, the next version should be a microservice with an IONIC chat and some more things. When it works, visiting JHipsterPress will explain what it is in itself.

**Programmers -like me- that are not good enough to help with Jhipster or earn a bug bounty might be interested in something easier to start with, like JHipsterPress.** At the same time, JHipsterPress could create a community of these type of programmers who are asking the same thing time and again to Jon at gitter, but would like to join let's say a IONIC community (or an ElasticSearch, Websocket communtiy) that are doing things in IONIC, ElasticSearch or Websocket and have a live project that uses IONIC as an example they can play with. Those projects should be open source for others to use them if the want to and should be running live projects. Well, it is longer to explain than this, but I'm sure you get the idea. 

When it works it looks like this: https://youtu.be/PelM_MyACkM

Let me get back to you when I finish this version and if you like it, you can create a new section, use another section or decide not to support it at all.

Thanks Deepu. 

I just need to finish it. I will get back to you when it is ready.

Reply:
ok let me know when it works, I'll review and let you know

Reply:
Hi Deepu,

There it is, working (to me, at least today, though I make changes):  admin/admin - user/user

So if you think it is worth to share it I'll be glad to work on the evolution of JhipsterPress and move it forward.

Thanks again and if you have any other idea, let me know.

Reply:
@deepu105 - it appears you had an extra "s" in the url when [you tried](https://github.com/jhipster/jhipster.github.io/pull/644#discussion_r238070219)

Regarding the 404 - here's what I saw - we get a 404 first _after a good few seconds_ and the page automatically refreshes/loads the site

Reply:
Thanks @vivekmore You are right. I will fix it.